var/spack/repos/builtin/packages/spdk/package.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Spdk(AutotoolsPackage):
+    """The Storage Performance Development Kit (SPDK) provides a set of tools
+    and libraries for writing high performance, scalable, user-mode storage
+    applications. It achieves high performance by moving all of the
+    necessary drivers into userspace and operating in a polled mode instead
+    of relying on interrupts, which avoids kernel context switches and
+    eliminates interrupt handling overhead.
+    """
+
+    homepage = "https://spdk.io"
+    url = "https://github.com/spdk/spdk/archive/v23.01.tar.gz"
+    git = "https://github.com/spdk/spdk"
+    maintainers("hyoklee")
+
+    version("master", branch="master", submodules=True)
+    version("23.01", tag="v23.01", submodules=True)
```

Comment:
This is the version used by DAOS 2.2, see: https://github.com/daos-stack/daos/blob/v2.2.0/utils/build.config
```suggestion
    version("22.01.1", tag="v22.01.1", submodules=True)
```